### ISSUE TITLE
The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.

### ISSUE THREAD

**Issue by aspatic** [multimodal-art-projection/YuE#92]
I was trying to run this model, but the inference of stage 1 stuck at 0% forever and I get this prompt:

![Image](https://github.com/user-attachments/assets/1f6ac4bf-df54-4277-ae5e-fe3a8fa142b6)

**Comment by a43992899** [comment]
You can ignore the warning and wait longer. What's your hardware?

**Comment by aspatic** [comment]
V100 16G, 8 cards in a node.
I guess its e-waste now.

**Comment by Orionx3000** [comment]
> You can ignore the warning and wait longer. What's your hardware?

I am running a 3080 12g vram and I have rendered over 2 albums of music with this program. I lost my c drive to a crash and reinstall and I am getting this error... It's not hardware and I waited hours didn't  do jack sht
